internal/csi-addons/networkfence/fencing.go

```diff
@@ -91,8 +99,21 @@ func (nf *NetworkFence) addCephBlocklist(ctx context.Context, ip string) error {
 // AddNetworkFence blocks access for all the IPs in the IP range mentioned via the CIDR block
 // using a network fence.
 func (nf *NetworkFence) AddNetworkFence(ctx context.Context) error {
+	hasBlocklistRangeSupport := true
 	// for each CIDR block, convert it into a range of IPs so as to perform blocklisting operation.
 	for _, cidr := range nf.Cidr {
+		// try range blocklist cmd, if invalid fallback to
+		// iterating through IP range.
+		if hasBlocklistRangeSupport {
+			err := nf.addCephBlocklist(ctx, cidr, true)
```

Comment:
instead of true and false pass hasBlocklistRangeSupport

Reply:
pass it where ? 

We would still need to parse and loop through the range in this layer.

Reply:
instead of passing true to `addCephBlocklist` pass `hasBlocklistRangeSupport`